forgettable-frontend/src/components/PersonCard/PersonCard.module.css

```diff
@@ -62,7 +62,7 @@
     font-size: var(--text-card-body-large);
     /* line-height: 0; */
```

Comment:
Should we remove the commented out "line-height"?

Reply:
Yep this will be fixed as well!